core/internal/runconsolelogs/filestreamwriter.go

```diff
@@ -11,34 +10,24 @@ import (
 // filestreamWriter sends modified console log lines to the filestream.
 type filestreamWriter struct {
 	FileStream filestream.FileStream
+
+	// StructuredLogs indicates whether to send structured logs.
```

Comment:
comment doesn't add much beyond the field name


```suggestion
	// Structured controls the format of uploaded lines.
	//
	// If false, then the legacy plain text format is used:
	//
	// ERROR 2023-04-05T10:15:30.000000 [stderr] Critical error occurred
	//
	// If true, then the newer JSON format is used:
	//
	//  {
	//    "level":"error",
	//    "ts":"2023-04-05T10:15:30.000000",
	//    "label":"node-1-rank-0",
	//    "content":"Division by zero",
	//  }
	Structured bool```